.github/workflows/release.yml

```diff
@@ -7,10 +7,10 @@ jobs:
       image: openjdk:15-jdk
       options: --user root
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - uses: gradle/wrapper-validation-action@v1
       - run: ./gradlew build publish --stacktrace
         env:
           MAVEN_URL: ${{ secrets.MAVEN_URL }}
           MAVEN_USERNAME: ${{ secrets.MAVEN_USERNAME }}
-          MAVEN_PASSWORD: ${{ secrets.MAVEN_PASSWORD }}
\ No newline at end of file
+          MAVEN_PConstantPoolASSWORD: ${{ secrets.MAVEN_PASSWORD }}
```

Comment:
Interesting choice of environment property name

Reply:
ooh yeah, idk how that happened 👍